challenges/objects-arrays.js

```diff
@@ -59,11 +83,19 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (i = 0; i<graduates.length; i++) {
+  contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (i = 0; i<graduates.length; i++) {
+  if (graduates[i].university.includes('Uni') === true) {
```

Comment:
.includes will return true or false and the if statement will only go on based on a true, so you don't actually have to  use the `=== true`, you can just say `if(true) {dostuff}` or `if (graduates[i].university.includes('Uni')) {dostuff}`